src/k8s/pkg/k8sd/api/cluster_config.go

```diff
@@ -53,6 +52,13 @@ func (e *Endpoints) putClusterConfig(s *state.State, r *http.Request) response.R
 		return response.InternalError(fmt.Errorf("database transaction to update cluster configuration failed: %w", err))
 	}
 
+	// Trigger an update of the configuration.
+	// Do not wait if the channel is full. The reconcilation loop will apply the most recent changes
+	select {
+	case e.provider.UpdateNodeConfigurationControllerCh() <- struct{}{}:
+	default:
+	}
+
```

Comment:
we should notify the controller at the end, there's more configuration updates later in this function which might be lost

Reply:
changed to `NotifyUpdateConfigMap` good idea!